base/operators.jl

```diff
@@ -5,9 +5,18 @@
 """
     <:(T1, T2)
 
-Subtype operator: returns `true` if and only if all values of type `T1` are
+Subtype predicate: returns `true` if and only if all values of type `T1` are
 also of type `T2`.
 
+Julia is sometimes not able to prove a subtyping relation that would ideally
```

Comment:
the word "ideally" here feels a little under-specified. which ideal? I would reword this to be a little more direct with less implications about some hypothetical mental model of types: something more like

> Subtype predicate: returns `true` if and only if all values of type `T1` are
also of type `T2`.

> This operator returns `true` when inference is able to prove a subtyping relationship, and may return `false` otherwise.


aka focusing on what it _does_ do not what it doesn't